challenges-webapp/src/main/webapp/skin/less/challenges.less

```diff
@@ -1,10 +1,13 @@
 @import "variables.less";
 @import "mixins.less";
 
-#ChallengesApplication {
+#EngagementCenterApplication {
```

Comment:
You may not need this change, please the ID `ChallengesApplication` inside `Challenges.vue`

Reply:
I have to do it because I have to set the margin for the whole interface, I put challenge-applications for the challenge style

Reply:
as discussed, this should be managed to keep the exact structure that was before by applying the ID on the newly introduced parent element (`div` instead of `v-app`) inside `Challenges.vue`. This will avoid to change the HTML structure of the page. For the margin, it would be better to apply it on the toolbar so that it's added only when the toolbar is visible (Feature flag enabled).